client/src/screens/training/controller.js

```diff
@@ -6,22 +6,25 @@ export const controller = (serviceLocator) => {
   const getParams = () => navigationApi.getPathParams(
     navigationApi.routes.openTraining,
   );
+  const createSetsPreview = (sets) => {
+    let strSets = '';
```

Comment:
it's a bad practice to write type of data in variable name (I'm about `str`).
Better to name more abstract, because the type inside of variable might be changed.
Think about, for example, `let preview = ''`